### PR TITLE
:sparkles:feat(llm): trustchain integration to qr code scan

### PR DESCRIPTION
**Pull request by LucasWerey**
### ✅ Checklist

- [x] `npx changeset` was attached.
- [ ] **Covered by automatic tests.** 
- [x] **Impact of the changes:** 
  - LLM WS flow 

### 📝 Description

Add the trustchain integration to the qr code scanned for WS
This allow us to import accounts using WS scanning LLD qr code with LLM

After a scan => pin code : 
 - if wrong => error screen on LLD and LLM
 - if good => activation screen on LLD and LLM

### ❓ Context

- **JIRA or GitHub link**: [LIVE-12168](https://ledgerhq.atlassian.net/browse/LIVE-12168)

---

### 🧐 Checklist for the PR Reviewers

- **The code aligns with the requirements** described in the linked JIRA or GitHub issue.
- **The PR description clearly documents the changes** made and explains any technical trade-offs or design decisions.
- **There are no undocumented trade-offs**, technical debt, or maintainability issues.
- **The PR has been tested** thoroughly, and any potential edge cases have been considered and handled.
- **Any new dependencies** have been justified and documented.
- **Performance** considerations have been taken into account. (changes have been profiled or benchmarked if necessary)

[LIVE-12168]: https://ledgerhq.atlassian.net/browse/LIVE-12168?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: apps/ledger-live-mobile/src/newArch/features/WalletSync/components/Activation/ActivationFlow.tsx [hunk 1]
```diff
@@ -32,7 +35,21 @@ const ActivationFlow = ({
...
+
+  const { handleStart, handleSendDigits, inputCallback, digits } = useSyncWithQrCode();
+
+  const handleQrCodeScanned = (data: string) => {
+    onQrCodeScanned();
+    if (memberCredentials) handleStart(data, memberCredentials, setCurrentStep);
+  };
+
+  const handlePinCodeSubmit = (input: string) => {
+    if (input && inputCallback && digits === input.length) handleSendDigits(inputCallback, input);
```
Comment (mcayuelas-ledger) [inline apps/ledger-live-mobile/src/newArch/features/WalletSync/components/Activation/ActivationFlow.tsx:50]: should be ` digits.length === input.length` no?

**Review by mcayuelas-ledger — DISMISSED** [review]

File: apps/ledger-live-mobile/src/newArch/features/WalletSync/hooks/useSyncWithQrCode.ts [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+    async (
+      url: string,
+      memberCredentials: MemberCredentials,
+      setCurrentStep: (step: Steps) => void,
+    ) => {
+      try {
+        const trustchain = await createQRCodeCandidateInstance({
+          memberCredentials,
+          scannedUrl: url,
+          memberName: Platform.OS + " " + Platform.Version,
```
Comment (gre) [inline apps/ledger-live-mobile/src/newArch/features/WalletSync/hooks/useSyncWithQrCode.ts:51]: in `hooks/useTrustchainSdk` we do

```
    const name = `${platformMap[Platform.OS] ?? Platform.OS} ${Platform.Version} ${hash ? " " + hash : ""}`;
```

can we factorize and use the same code? thanks

**Review by gre — CHANGES_REQUESTED** [review]

File: apps/ledger-live-mobile/src/newArch/features/WalletSync/hooks/useSyncWithQrCode.ts (hunk 2 above)
Comment (mcayuelas-ledger) [reply apps/ledger-live-mobile/src/newArch/features/WalletSync/hooks/useSyncWithQrCode.ts:51]: Indeed Nice catch

Comment (LucasWerey) [reply apps/ledger-live-mobile/src/newArch/features/WalletSync/hooks/useSyncWithQrCode.ts]: Done ✅

**Review by gre — APPROVED** [review]

**Pull request merged**